test/original/BitTorrent-1.0.txt

```diff
@@ -0,0 +1,71 @@
+      f. Distribution of Derivative Works. You may create Derivative Works (e.g., combinations of some or all of the Licensed Product with other code) and distribute the Derivative Works as products under any other license you select, with the proviso that the requirements of this License are fulfilled for those portions of the Derivative Works that consist of the Licensed Product or any Modifications thereto.
+      g. Compensation for Distribution of Executable Versions of Licensed Products, Modifications or Derivative Works. Notwithstanding any provision of this License to the contrary, by distributing, selling, licensing, sublicensing or otherwise making available any Licensed Product, or Modification or Derivative Work thereof, you and Licensor hereby acknowledge and agree that you may sell, license or sublicense for a fee, accept donations or otherwise receive compensation for executable versions of a Licensed Product, without paying a royalty or other fee to the Licensor or any other Contributor, provided that such executable versions (i) contain your or another Contributor's material Modifications, or (ii) are otherwise material Derivative Works. For purposes of this License, an executable version of the Licensed Product will be deemed to contain a material Modification, or will otherwise be deemed a material Derivative Work, if (a) the Licensed Product is modified with your own or a third party's software programs or other code, and/or the Licensed Product is combined with a number of your own or a third party's software programs or code, respectively, and (b) such software programs or code add or contribute material value, functionality or features to the License Product. For the avoidance of doubt, to the extent your executable version of a Licensed Product does not contain your or another Contributor's material Modifications or is otherwise not a material Derivative Work, in each case as contemplated herein, you may not sell, license or sublicense for a fee, accept donations or otherwise receive compensation for such executable. Additionally, without limitation of the foregoing and notwithstanding any provision of this License to the contrary, you cannot charge for, sell, license or sublicense for a fee, accept donations or otherwise receive compensation for the Source Code.
```

Comment:
I think we need to get to consensus on this (or decide that we don't need to) before merging the CI tests against known licenses. So far my impression has been that it's just be @goneall and @wking, but hopefully getting some more technical input can help us get to a decision here.


I don't know enough to say for sure which way I'd prefer, nor would I say that my opinion is worth much here (as I am not fully informed), but on the current evidence I think I would lean toward keeping the original texts for testing in this same repo (the idea being that if you PR to add a license, that PR should also contain the original text). From what I can glean from the issues firehose, that seems more like @goneall's position than @wking. But I do *not* think I should be the deciding vote on this issue.

Reply:
> … evidence I think I would lean toward keeping the original texts for testing in this same repo (the idea being that if you PR to add a license, that PR should also contain the original text).

But do you expect the legal team to review the test text being added in this repo?  I think that review would be more likely without the noise like my [question about Bash vs. vanilla POSIX][1] and similar.  And if you *do* expect legal-team review of test-text being added here, do you also expect there to be legal-team review when that same (hopefully ;) content is submitted to license-test-files (e.g. with spdx/license-test-files#6)?  That seems like twice the work to me.

[1]: https://github.com/spdx/license-list-XML/pull/592#discussion_r159300232

Reply:
We did discuss this on one of the legal calls and there was consensus for those attending the call that we could add the license text to the license-list-XML repository.

I don't expect we will gain consensus on this topic and I do not want to be prevented from automating the testing due to the lack of consensus.

We can always change the location of the tests in a future release.

Reply:
Totally fair -- I didn't recall having addressed this on a call.

Reply:
> I don't expect we will gain consensus on this topic…

Can we at least reach a narrower consensus around BitTorrent-1.0's 4.g being broken or not?  I'd really like to avoid committing known-broken test data to this repository.